ci/qemu/preseed.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
What is this file for?

Reply:
It's explained in the readme on that folder